src/olympia/ratings/migrations/0009_alter_deniedratingword_word.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.6 on 2023-10-27 10:17
+
+from django.db import migrations, models
+import olympia.ratings.models
+
+
```

Comment:
What triggered this migration?

Reply:
Catching up on `ratings` migrations. I don't have to include it, but it's a no-op.